api/ccapi/include/tensor_dim.h

```diff
@@ -103,7 +187,17 @@ class TensorDim {
    * @param shape shape of format
    * @param fm format NCHW | HNWC
    */
-  TensorDim(const std::string &shape, Format fm = Format::NCHW);
+  TensorDim(const std::string &shape, TensorType t_type_ = TensorType());
+
+  /**
+   * @brief Construct a new Tensor Dim object
+   *
+   * @param shape shape of format
+   * @param fm format NCHW | HNWC
+   * @param d_type data type FP16 | FP32
+   */
+  TensorDim(const std::string &shape, TensorDim::Format fm,
```

Comment:
maybe pass default argument for `fm`
```suggestion
  TensorDim(const std::string &shape, TensorDim::Format fm = TensorDim::Format::NCHW,
```